Pranaystwt.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "Pranaystwt",
+  "url": "https://github.com/PranaySewatkar",
+  "image": "https://github.com/PranaySewatkar.png",
+  "why": "To be consistent with myself",
+  "weekly": [50 incline pushup, 50 jumping jack, walk daily, stretch daily],
```

Can you please wrap the weekly items inside`"`
```suggestion
  "weekly": ["50 incline pushup", "50 jumping jack", "walk daily", "stretch daily"],
```